utils/linting/husky.config.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+// https://github.com/typicode/husky#supported-hooks
+const config = {
+  'pre-commit': 'npm run lint && git add .', // run the lint, and include any auto fixes
```

Comment:
Is this running risk of adding everything in repo when you might not want to? https://github.com/okonet/lint-staged might be a better answer

Reply:
It would indeed add everything to the commit in it's current state. I will look into lint-staged and update as needed